autoload/lsp/completion.vim

```diff
@@ -135,6 +137,26 @@ def CompletionUltiSnips(prefix: string, items: list<dict<any>>)
   endfor
 enddef
 
+# add completion from current buf
+def CompletionFromBuffer(items: list<dict<any>>)
+    var words = {}
+    for line in getline(1, '$')
+        for word in line->split('\W\+')
+            if !has_key(words, word) && len(word) > 1
```

Comment:
Can you change this to `if !words->has_key(word) && word->len() > 1`?

Reply:
done in https://github.com/yegappan/lsp/pull/284/commits/a7b2ba9f3f7f9d356f4cca8f38b834a62790444d